.github/workflows/deploy-lambda-assertion.yml

```diff
@@ -0,0 +1,68 @@
+name: Deploy Lambda assertion
+
+on:
+    push:
+      branches:
+        - "main"
+      paths:
+        - "**/src/oneid/oneid-lambda-assertion/**"
+        - "**/src/oneid/oneid-common/**"
+    workflow_dispatch:
+      inputs:
+        environment:
+          description: 'Choose environment'
+          type: choice
+          required: true
+          default: dev
+          options:
+          - dev
+jobs:
+    build:
+      runs-on: ubuntu-22.04
+
+      steps:
+      - name: Checkout code
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'  # Specify the Python version
+      
+      - name: Install dependencies
+        run: pip install -r requirements.txt
+        with:
+           path: ./src/oneid/oneid-lambda-assertion/requirements.txt
+        
+      - name: Archive build artifacts
+        uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808
```

Comment:
Does this upload artifact also produce the zip file. Could it be it just upload the file?

Reply:
In the repo for upload-artifact is described that "When an Artifact is uploaded, all the files are assembled into an immutable Zip archive. There is currently no way to download artifacts in a format other than a Zip or to download individual artifact contents."